packages/react/src/components/LightboxMediaViewer/LightboxMediaViewer.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ExpressiveModal } from '../ExpressiveModal';
+import { ModalBody } from 'carbon-components-react';
+import { settings } from 'carbon-components';
+import { DDS_LIGHTBOX_MEDIA_VIEWER } from '../../internal/FeatureFlags';
+import {
+  featureFlag,
+  settings as ddsSettings,
+} from '@carbon/ibmdotcom-utilities';
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * LightboxMediaViewer Component
+ *
+ * @param {object} props props object
+ * @param {string} props.title LightboxMediaViewer media title
+ * @param {string} props.copy LightboxMediaViewer media short description
+ * @param {string} props.image LightboxMediaViewer responsive image object
+ * @param {boolean} props.open sets whether the modal is open/close
+ * @param {Function} props.onClose do something on close in addition, return false to completely replace
+ * @returns {*} JSX Object
+ */
+const LightboxMediaViewer = ({ title, copy, image, open, onClose }) => {
+  /**
+   * Set default image
+   *
+   * @param {*} image responsive image object
+   * @returns {string} default image url
+   */
+  const defaultImage = image => {
+    return image.uri.md
+      ? image.uri.md
+      : image.uri.lg
+      ? image.uri.lg
+      : image.uri.sm;
+  };
+
+  if (!image.uri.lg) {
+    return null;
+  }
+  return featureFlag(
+    DDS_LIGHTBOX_MEDIA_VIEWER,
+    <section
```

Comment:
Not sure if a section is necessary. This is in a lot of ways just a wrapper for the `ExpressiveModal` with a very specific layout and data model. We should be able to just pass in the `className` without a problem. I wonder if it makes sense to override the `data-autoid` for the `ExpressiveModal` since it's technically different, either that or just keep it the same, and `LightboxMediaViewer` doesn't get a new a specific stable selector because its shared. 🤔idk... @jeffchew any thoughts?

```jsx
return (
  <ExpressiveModal
    data-autoid={`${stablePrefix}--lightbox`}
    className={`${prefix}--lightbox`}>
    <MediaViewer />
  </ExpressiveModal>
);
```

Reply:
@photodow @sannminwin I would say keep it the same.